conf/ci-minimal/dialplan/switchydp.xml

```diff
@@ -4,6 +4,7 @@
   <!-- Park call and transfer control to esl -->
   <extension name="switchiopark">
     <condition field="destination_number" expression="^(.*)$">
+      <action application="set" data="park_timeout=3:DESTINATION_OUT_OF_ORDER"/>
```

Comment:
The NETWORK_OUT_OF_ORDER cause seems more appropriate (is not a particular upstream destination number/uri that is out of order). As an add-on benefit that cause code also returns a SIP 503 which could be complemented by a 'Retry-After' header (setting variable sip_rh_Retry-After [0]) suggesting to retry after X number of secs to throttle down requests if possible and desired.

[0] https://wiki.freeswitch.org/wiki/Sofia-SIP#Adding_Response_Headers

Reply:
Be aware this timeout might have unintended side effects. See my comments on issue #47.

Reply:
@moises-silva nice! I'll change it.